enterprise/glassfish.javaee/src/org/netbeans/modules/glassfish/javaee/Hk2JpaSupportImpl.java

```diff
@@ -97,10 +109,12 @@ private static class JpaSupportVector {
     static {
         for (GlassFishVersion version : GlassFishVersion.values()) {
             jpaSupport[version.ordinal()] = new JpaSupportVector(
-                    GlassFishVersion.lt(version, GlassFishVersion.GF_6),
-                    GlassFishVersion.lt(version, GlassFishVersion.GF_6) && GlassFishVersion.ge(version, GlassFishVersion.GF_3),
-                    GlassFishVersion.lt(version, GlassFishVersion.GF_6) && GlassFishVersion.ge(version, GlassFishVersion.GF_4),
-                    GlassFishVersion.lt(version, GlassFishVersion.GF_6) && GlassFishVersion.ge(version, GlassFishVersion.GF_5)
+                    GlassFishVersion.lt(version, GlassFishVersion.GF_7_0_0),
+                    GlassFishVersion.lt(version, GlassFishVersion.GF_7_0_0) && GlassFishVersion.ge(version, GlassFishVersion.GF_3),
+                    GlassFishVersion.lt(version, GlassFishVersion.GF_7_0_0) && GlassFishVersion.ge(version, GlassFishVersion.GF_4),
+                    GlassFishVersion.lt(version, GlassFishVersion.GF_7_0_0) && GlassFishVersion.ge(version, GlassFishVersion.GF_5),
```

Comment:
JPA 1.0, 2.0, 2.1 and 2.2 are not supported by GlassFish 6 and upward, so the comparison for JPA below 3.0  should be against `GlassFishVersion.GF_6`